saos-webapp/src/main/java/pl/edu/icm/saos/webapp/judgment/detail/JudgmentDetailController.java

```diff
@@ -20,14 +21,18 @@
 	@Autowired
 	private JudgmentRepository judgmentRepository;
 	
+	@Autowired
+	private JudgmentCorrectionService judgmentCorrectionService;
+	
 	
 	//------------------------ LOGIC --------------------------
 	
 	@RequestMapping("/judgments/{judgmentId}")
 	public String showJudgmentDetail(ModelMap model, @PathVariable("judgmentId") Integer judgmentId) {
```

Comment:
showJudgmentDetail -> pokaż szczegół orzeczenia
showJudgmentDetails -> pokaż szczegóły orzeczenia
Consider the name change.